bench_runner/templates/benchmark.src.yml

```diff
@@ -31,6 +31,10 @@ run-name: benchmarking ${{ inputs.fork }}/${{ inputs.ref }}
         description: "Also collect pystats for this commit"
         type: boolean
         default: false
+      publish:
+        description: "Publish results to the corresponding public repo"
+        type: boolean
+        default: true
```

Comment:
This doesn't need to be here any more -- we don't want to make this a "run time" configuration.